frictionless/steps/field/field_merge.py

```diff
@@ -0,0 +1,104 @@
+from ...step import Step
+from ...field import Field
+from typing import List, Iterator
+from petl.compat import next, text_type
+
+
+def merge(
```

Comment:
We need to put low-level functions (internal) at the bottom while having high-level function (API) at the top:
- field_merge
- merge
Similar for the second step. It's better because if someone reads the code he first needs to see high-level functionality etc while low-level implementation might be not so important